tests/simplify_sitelinks.ts

```diff
@@ -1,6 +1,6 @@
 // @ts-nocheck
```

Comment:
this is dangerous. SimplifiedSitelinks seems to have the wrong type

Reply:
Yeah, there are still a bunch of these left. Some I could remove or easily replace by `ts-expect-error`s

Before we had only JavaScript without types so only having a few of these is already a huge improvement. And we will get there :)